packages/react-components/theme-designer/src/components/Export/ExportPanel.tsx

```diff
@@ -0,0 +1,185 @@
+  },
+});
+
+export const ExportPanel = () => {
+  const {
+    dispatch,
+    state: { showExportPanel, themeName, brand, lightThemeOverrides, darkThemeOverrides },
+  } = useThemeDesigner();
+
+  const onCloseExportPanel = () => {
+    dispatch({ type: 'showExportPanel', payload: false });
+  };
+
+  const styles = useStyles();
+
+  const [selectedValue, setSelectedValue] = React.useState<TabValue>('Code');
+
+  const onTabSelect = (event: SelectTabEvent, data: SelectTabData) => {
+    setSelectedValue(data.value);
+  };
+
+  const codeValue = dedent`
+  const ${themeName}: BrandVariants = { ${objectToString(brand, '\u00A0\u00A0')} };
+
+  const lightTheme: Theme = {
+    ...createLightTheme(${themeName}), ${getBrandValues(brand, lightThemeOverrides, themeName, '\u00A0\u00A0')} };
+
+  const darkTheme: Theme = {
+    ...createDarkTheme(${themeName}), ${getBrandValues(brand, darkThemeOverrides, themeName, '\u00A0\u00A0')} };
+  
+
+  darkTheme.colorBrandForeground1 = ${themeName}[110];
+  darkTheme.colorBrandForeground2 = ${themeName}[120];
+  `;
+
+  const jsonLightValue = dedent`
+   ${JSON.stringify({ ...createLightTheme(brand), ...lightThemeOverrides }, null, '\t')}`;
+
+  const jsonDarkValue = dedent`
+     ${JSON.stringify(
+       {
+         ...createDarkTheme(brand),
+         ...{ colorBrandForeground1: brand[110], colorBrandForeground2: brand[120] },
+         ...darkThemeOverrides,
+       },
+       null,
+       '\t',
+     )} 
+  `;
+
+  const exportedValue = React.useMemo(() => {
+    switch (selectedValue) {
+      case 'Code':
+        return codeValue;
+      case 'JSONLight':
+        return jsonLightValue;
+      case 'JSONDark':
+        return jsonDarkValue;
+      default:
+        return '';
+    }
+  }, [codeValue, jsonLightValue, jsonDarkValue, selectedValue]);
+
+  const onClickCopyToClipboard = () => {
+    navigator.clipboard.writeText(exportedValue);
+  };
+
+  return (
+    <>
+      {showExportPanel && (
+        <FluentProvider theme={webLightTheme}>
+          <div
+            style={{
+              zIndex: 100,
+              position: 'absolute',
+              top: '0px',
+              right: '0px',
+              width: '400px',
+              border: `1px solid ${tokens.colorNeutralStroke1}`,
+              borderRadius: tokens.borderRadiusXLarge,
+              backgroundColor: tokens.colorNeutralBackground1,
+              boxShadow: tokens.shadow64,
+            }}
+          >
+            <div style={{ margin: '16px' }}>
+              <div className={styles.exportHeader}>
+                <Text as="h1" id="headingID" size={500}>
+                  Export Theme
+                </Text>
+                <Button
+                  size="small"
+                  appearance="subtle"
+                  icon={<DismissSquare24Regular />}
+                  // eslint-disable-next-line react/jsx-no-bind
+                  onClick={onCloseExportPanel}
+                />
+              </div>
```

Comment:
Looks like this button is getting a little messed up:
![image](https://github.com/microsoft/fluentui/assets/5953191/358fb46c-0894-4fb1-ad93-d89fd4010465)

Adding `align-items: center` to the `exportHeader` classname and `height: max-content` to the button's classname should fix it.

Reply:
Thanks for that suggestion! Changed to "center". Height suggestion to button didn't add anything so didn't add.